app/assets/stylesheets/layout/_profile.scss

```diff
@@ -4,27 +4,42 @@
 .profile {
   background: $color-ice-white;
   padding-top: 35px;
+  .panel {
+    border: 1px solid #b3b1a5;
```

Comment:
Color literals like `#b3b1a5` should only be used in variable declarations; they should be referred to via variable everywhere else.